e2e/transactions/transactions_test.go

```diff
@@ -224,91 +225,7 @@ func (s *TransactionsTestSuite) TestSendContractTx() {
-
-	// replace transaction notification handler
-	var txHash = gethcommon.Hash{}
-	signal.SetDefaultNodeNotificationHandler(func(jsonEvent string) { // nolint: dupl
-		var envelope signal.Envelope
-		err := json.Unmarshal([]byte(jsonEvent), &envelope)
-		s.NoError(err, fmt.Sprintf("cannot unmarshal JSON: %s", jsonEvent))
-
-		if envelope.Type == txqueue.EventTransactionQueued {
-			event := envelope.Event.(map[string]interface{})
-			log.Info("transaction queued (will be completed shortly)", "id", event["id"].(string))
-
-			// the first call will fail (we are not logged in, but trying to complete tx)
-			log.Info("trying to complete with no user logged in")
-			txHash, err = s.Backend.CompleteTransaction(
-				common.QueuedTxID(event["id"].(string)),
-				TestConfig.Account1.Password,
-			)
-			s.EqualError(
-				err,
-				account.ErrNoAccountSelected.Error(),
-				fmt.Sprintf("expected error on queued transaction[%v] not thrown", event["id"]),
-			)
-
-			// the second call will also fail (we are logged in as different user)
-			log.Info("trying to complete with invalid user")
-			err = s.Backend.AccountManager().SelectAccount(sampleAddress, TestConfig.Account1.Password)
-			s.NoError(err)
-			txHash, err = s.Backend.CompleteTransaction(
-				common.QueuedTxID(event["id"].(string)), TestConfig.Account1.Password)
-			s.EqualError(
-				err,
-				txqueue.ErrInvalidCompleteTxSender.Error(),
-				fmt.Sprintf("expected error on queued transaction[%v] not thrown", event["id"]),
-			)
-
-			// the third call will work as expected (as we are logged in with correct credentials)
-			log.Info("trying to complete with correct user, this should succeed")
-			s.NoError(s.Backend.AccountManager().SelectAccount(TestConfig.Account1.Address, TestConfig.Account1.Password))
-			txHash, err = s.Backend.CompleteTransaction(
-				common.QueuedTxID(event["id"].(string)),
-				TestConfig.Account1.Password,
-			)
-			s.NoError(err, fmt.Sprintf("cannot complete queued transaction[%v]", event["id"]))
-
-			log.Info("contract transaction complete", "URL", "https://ropsten.etherscan.io/tx/"+txHash.Hex())
-			close(completeQueuedTransaction)
-			return
-		}
-	})
-
-	// this call blocks, up until Complete Transaction is called
-	txHashCheck, err := s.Backend.SendTransaction(context.TODO(), common.SendTxArgs{
-		From:  common.FromAddress(TestConfig.Account1.Address),
-		To:    common.ToAddress(TestConfig.Account2.Address),
-		Value: (*hexutil.Big)(big.NewInt(1000000000000)),
-	})
-	s.NoError(err, "cannot send transaction")
-
-	select {
-	case <-completeQueuedTransaction:
-	case <-time.After(2 * time.Minute):
-		s.FailNow("completing transaction timed out")
-	}
-
-	s.Equal(txHashCheck.Hex(), txHash.Hex(), "transaction hash returned from SendTransaction is invalid")
-	s.False(reflect.DeepEqual(txHashCheck, gethcommon.Hash{}), "transaction was never queued or completed")
-	s.Zero(s.Backend.TxQueueManager().TransactionQueue().Count(), "tx queue must be empty at this point")
-}
```

Comment:
This function as not needed any more as we now only run tests again StatusChain.

Reply:
Will it be needed later again, after #424?

Reply:
Nope. Only one small change will be needed to ensure the node is synced which I've just added :)